utils/run_in_ga.sh

```diff
@@ -54,7 +54,5 @@ echo ::group::Setting up daos_admin
 echo ::endgroup::
 
 echo ::group::Container copy test
-# DAOS-7440
-export LD_LIBRARY_PATH=/opt/daos/prereq/release/spdk/lib/
```

Comment:
But this was required last time I tested it, so I think we can say this PR closes the ticket.

Reply:
This is inexplicable to me but I'm happy to close it